symfony/config/packages/security.yaml

```diff
@@ -1,24 +1,56 @@
 security:
+    encoders:
+        App\Entity\User: bcrypt
+
     # https://symfony.com/doc/current/security.html#where-do-users-come-from-user-providers
     providers:
-        users_in_memory: { memory: null }
+        # used to reload user from session & other features (e.g. switch_user)
+        app_user_provider:
```

Comment:
Ik denk dat we eigen UserProvider gaan moeten schrijven aangezien we meer velden hebben dan username/email en password. Fields zoals deviceId, deviceToken moeten zeker al voorzien worden voor notifications.